README.md

```
@@ -0,0 +1,34 @@
+Vize Randevu Takip Botu
+
+Bu Python betiği, Amerika Birleşik Devletleri'ne yapılan vize başvurularının randevu tarihlerindeki değişiklikleri izlemek için kullanılır.
+Belirli bir web sitesindeki vize randevu sayfasını sürekli olarak kontrol eder ve yeni randevu tarihleri mevcut olduğunda, Telegram botu aracılığıyla bildirim gönderir.
+
+### Özellikler:
+- Web sitesindeki randevu tarihlerini periyodik olarak kontrol eder.
+- Yeni randevu tarihleri bulunduğunda, Telegram botu aracılığıyla bildirim gönderir.
+- Hata durumunda tekrar denemek için 10 dakika bekler.
+- Her 5 dakikada bir vize randevusu sayfası kontrol edilir.
+
+### Gereksinimler:
```

Markdownlint (0.37.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)